assets/large_language_models/dbcopilot/environments/dbcopilot_embeddings/context/Dockerfile

```diff
@@ -53,7 +56,7 @@ RUN /bin/bash -c "source /opt/miniconda/etc/profile.d/conda.sh && \
     pip install --upgrade pip && \
     pip install  -r requirements.txt && \
     pip install  db-copilot-tool==0.0.8 &&\
-    pip install  cryptography==43.0.1 langchain==0.1.11 idna==3.7 sqlparse==0.5.0 gunicorn==22.0.0 Werkzeug==3.0.3 requests==2.32.0 azure-identity==1.16.1"
+    pip install  cryptography==43.0.1 langchain==0.2.10 idna==3.7 sqlparse==0.5.0 gunicorn==22.0.0 Werkzeug==3.0.3 requests==2.32.0 azure-identity==1.16.1"
```

Comment:
you should not need cryptography, requests, idna packages there if you are maintaining dependencies properly. Please consider installing all additional dependencies from requirements.txt